web/src/lib/utils.ts

```diff
@@ -194,3 +194,9 @@ export const findLocale = (code: string | undefined) => {
     name: language?.name,
   };
 };
+
+export const asyncTimeout = (ms: number) => {
+  return new Promise((resolve) => {
+    setTimeout(resolve, ms);
+  });
+};
```

Comment:
There's a promise version of `setTimeout` you can import from `timers/promises`

Reply:
Ah, thanks! I didn't know about that one. Can we import Node built-ins on the client? I'm not sure whether Vite will bundle Node built-ins or not. I'll have to test later when I get a minute

Reply:
Oops, I think not. I get `[plugin:vite:resolve] Module "timers/promises" has been externalized for browser compatibility` followed by `RollupError: [vite-plugin-sveltekit-compile] "setTimeout" is not exported by "__vite-browser-external"`.